src/app/modules/home/components/alert/alert.component.scss

```diff
@@ -0,0 +1,11 @@
+.alert {
+    text-align: center;
+    margin: 1rem auto;
+    width: 80%;
+  &__danger {
+    color: red;
+  }
+  &__success {
+    color: green;
```

Comment:
I think, we should not use colors like 'green' and 'red' because of in different browsers they gonna display differently.You should create a separate scss file like '_variables.scss' in folder scss, move all color into variable to this file, and then import at the place, where you need that